contracts/script/GenerateUnitTestHashes.s.sol

```diff
@@ -14,7 +14,7 @@ contract GenerateHashes is Script {
     string deployConfigPath = "script/input/eigenda_deploy_config.json";
 
     // deploy all the EigenDA contracts. Relies on many EL contracts having already been deployed.
-    function run() external view {
+    function run() external pure {
```

Comment:
from @samlaf  #1472 

comment is wrong. Also is this script useful? Looks like its just console.logging, which I'm guessing was used to manually create test cases for some other script..? There's def a better way to do this...

Reply:
I only made this change to fix a compiler warning which blocks CI with the updated forge-std. I don't think fixing the comment would be in scope of this PR

Reply:
Why not? Just delete the comment, its clearly wrong and was copy-pasted from somewhere else. I think its fine to include this kind of small change as part of a larger PR; otherwise it'll never be done because we'll just forget.

Reply:
If that is the reason, I must insist even further that it belongs in another PR since this one will likely not be merged for a while since I anticipate it will be hard to get consensus on it. I will put the fix in [this PR](https://github.com/Layr-Labs/eigenda/pull/1482) which we should be able to merge right away, and remove this changeset from this PR entirely.